src/commands/backslash.rs

```diff
@@ -30,6 +30,7 @@ const HELP: &str = r###"
 Introspection
   (options: -v = verbose, -s = show system objects, -I = case-sensitive match)
   \d [-v] NAME             describe schema object
+  \ds                      describe the whole schema
```

Comment:
No long variant?

Reply:
`\describe-schema`. Well, we list long variants here inconsistently :(